packages/maskbook/src/plugins/Trader/UI/trending/SearchResultView.tsx

```diff
@@ -153,15 +156,9 @@ export function SearchResultView(props: SearchResultViewProps) {
     const canSwap = trending.coin.eth_address || trending.coin.symbol.toLowerCase() === 'eth'
     const swapTabIndex = dataProvider !== DataProvider.UNISWAP ? 3 : 1
     const fromToken = chainId === ChainId.Mainnet && coin.is_mirrored ? UST : createEtherToken(chainId)
-    const { decimals, name: tokenName, symbol } = coinDetailed ?? trending.coin
```

Comment:
decimals is the only thing we should use from coinDetailed since the name and symbol of coinDetailed is empty string.

![image](https://user-images.githubusercontent.com/63733714/104564774-4a794500-5686-11eb-90d4-ca608e415241.png)

![image](https://user-images.githubusercontent.com/63733714/104564852-62e95f80-5686-11eb-84a3-780275bdd8c2.png)